<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterBorneFontaineInterface</class>
 <widget class="QDialog" name="AjouterBorneFontaineInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AjouterBorneFontaineInterface</string>
  </property>
  <widget class="QLabel" name="lbDirection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Direction&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbVille">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Ville&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbArrondissement">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Arrondissement&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLatitude">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Latitude&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbNomTopo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Nom topographique&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLongitude">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Longitude&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbIdentifiant">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Identifiant de la borne&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="txtIdentifiant">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboDirection">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nord</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sud</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Est</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ouest</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>null</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txtNomTopo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="txtLongitude">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="maximum">
    <double>100000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="txtLatitude">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtVille">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtArrondissement">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSoumettre">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Soumettre</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnSoumettre</sender>
   <signal>clicked()</signal>
   <receiver>AjouterBorneFontaineInterface</receiver>
   <slot>validerChamps()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validerChamps()</slot>
 </slots>
</ui>
